features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+    
+    @gba
+    Scenario: met fields is gevraagd om volledigeNaam
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999990160                       |
+      | fields              | naam.volledigeNaam              |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam                                 | waarde      |
+      | voornamen                            | Hendrik Jan |
+      | adellijkeTitelPredicaat.code         | R           |
+      | adellijkeTitelPredicaat.omschrijving | ridder      |
+      | voorvoegsel                          | de          |
+      | geslachtsnaam                        | Brink       |
+
+  Rule: wanneer gevraagd wordt om aanhef, aanschrijfwijze of gebruikInLopendeTekst, dan worden geslacht, naam en naam van de partner geleverd
+    - de partner worden adellijkeTitelPredicaat, voorvoegsel en geslachtsnaam geleverd
+   
+    @gba
+    Abstract Scenario: met fields is gevraagd om <fields>
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999990160                       |
+      | fields              | <fields>                        |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam                                 | waarde      |
+      | voornamen                            | Hendrik Jan |
+      | adellijkeTitelPredicaat.code         | R           |
+      | adellijkeTitelPredicaat.omschrijving | ridder      |
+      | voorvoegsel                          | de          |
+      | geslachtsnaam                        | Brink       |
+      En heeft de response een persoon met de volgende gegevens
+      | naam                  | waarde |
+      | geslacht.code         | M      |
+      | geslacht.omschrijving | man    |
+      En heeft de response een persoon met de volgende 'partner' gegevens
+      | naam                                 | waarde  |
+      | adellijkeTitelPredicaat.code         | BS      |
+      | adellijkeTitelPredicaat.omschrijving | barones |
+      | voorvoegsel                          | van der |
+      | geslachtsnaam                        | Horn    |
+
+      Voorbeelden:
+      | fields                           |
+      | aanhef                           |
+      | aanschrijfwijze                  |
+      | gebruikInLopendeTekst            |
+      | adressering.aanhef               |
+      | aanschrijfwijze.naam             |
+      | aanschrijfwijze.aanspreekvorm    |
+      | adressering.aanschrijfwijze.naam |
+
+  Rule: wanneer gevraagd wordt om leeftijd, wordt geboorte.datum en overlijden.datum geleverd
```

Comment:
Uit deze rule maak ik op dat `overlijden.datum` altijd geleverd wordt maar dat lijkt me (ook gezien het volgende scenario) niet het geval. Maak er dus:

`Rule: wanneer gevraagd wordt om leeftijd, wordt geboorte.datum en evt. overlijden.datum geleverd`

als dat kan tenminste.

Reply:
ik voeg een extra regel onder de rule toe:
- overlijden.datum wordt alleen geleverd wanneer de persoon overleden is (wanneer er een waarde is voor overlijden.datum)